qemu/tests/cfg/smt_test.cfg

```diff
@@ -0,0 +1,45 @@
+- smt_test:
+    virt_test_type = qemu
+    type = smt_test
+    start_vm = no
+    kill_vm_on_error = yes
+    only x86_64, i386
+    only HostCpuVendor.amd
+    cpu_model_flags += ",enforce"
+    auto_cpu_model = no
+    smp = 2
+    vcpu_maxcpus = ${smp}
+    vcpu_threads = 2
+    vcpu_cores = 1
+    vcpu_sockets = 1
+    test_negative_thread = yes
+    check_flag_cmd = lscpu | grep Flags | awk -F ':'  '{print $2}'
+    Windows:
+        get_cores_cmd = wmic cpu get NumberOfCores /value
+        get_sockets_cmd = wmic cpu get SocketDesignation /value
+        x86_64:
+            install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
+        i386,i686:
+            install_path = "C:\Program Files\JAM Software\HeavyLoad"
+        install_cmd = "start /wait %s:\HeavyLoadSetup.exe /verysilent"
+    Linux:
+        get_threads_cmd = lscpu | grep "Thread(s) per core"
+        stress_args = --cpu 2 --vm 2 --io 4 --vm-bytes 1024
+        get_cpu_usage_cmd = top -1 -n 1 | grep Cpu
+    variants:
+        - EPYC:
+            cpu_model = EPYC
+            variants:
+                - @default:
+                    expected_threads = 2
+                - no_topoext:
+                    expected_threads = 1
+                    cpu_model_flags += ",-topoext"
+        - Opteron:
```

Comment:
Better name it `with_EPYC` and `with_Opteron`, to avoid possible name conflict(although it's fine for now, maybe in the future).